### DISCUSSION TITLE
Is it possible to change the custom domain in the middle of the paid plan?

### DISCUSSION THREAD

**Discussion by fu-sen** [domcloud/domcloud#15] [Q&A] [answered]
The paid plan of DOM Cloud Hosting is an annual contract. It is expected to be used with the domain on a yearly basis.

I thought there were cases where I wanted to change the custom domain I was setting up.
Is that possible? Or do I need to sign up for a separate paid plan?
If possible, can I go through the dash boat immediately? Or do I need to use something like here or GitHub Issue?

**Comment by willnode** [comment, marked as answer]
Yes, it is possible to change the custom domain for any Hosting, you just need to do it in the "Change" button under "Domain Hosting" in the portal dashboard. There's no restriction on how much often you able to change the custom domain name.

Although you may already know this, you can't change the domain name of the domain you've bought (if you ever decide to buy a domain from here, anyway). I need to clarify this if someone else reading here, as essentially hosting and domain purchases are separate things internally.

**Reply by fu-sen** [reply]
Since you and I have talked about custom domains in the past, you will know that I own some domains.
I'm carefully considering a paid plan and the answer sounds good. Thank you. 😄

**Reply by willnode** [reply]
I've added [some information](https://github.com/domcloud/domcloud-io/commit/49688d7474b9c851e1007ce14bab0c27b9707c7a) in FAQ recently, that might be relevant.

**Reply by fu-sen** [reply]
![スクリーンショット 2021-07-15 12 38 11](https://user-images.githubusercontent.com/5434159/125725411-effeafb5-b266-49bf-98df-5df1400f1688.jpg)
Yeah, I'm seeing a `change domain` in the updated menu.
